src/sdk/model/company/company.ts

```diff
@@ -846,4 +874,172 @@ export class Company extends Entity {
         return new CompanyLocationBackupStatus(json);
       });
   }
+
+  /**
+   * Get the identity provider for the current company.
+   * @return {Promise<IdentityProvider>} provider The identity provider
```

Comment:
remove the first 'provider'. sort of looks like a param reference, not needed.

Reply:
Fixed